src/SingleProject/Resizetizer/src/GeneratePackageAppxManifest.cs

```diff
@@ -202,34 +207,26 @@ void UpdateManifest(XDocument appx)
 					{
 						var xname = "DisplayName";
 						var attr = visual.Attribute(xname);
-						if (attr == null || string.IsNullOrEmpty(attr.Value))
+						if (attr == null || string.IsNullOrEmpty(attr.Value) || attr.Value == DefaultPlaceholder)
 							visual.SetAttributeValue(xname, ApplicationTitle);
 					}
 
 					// Description=""
 					{
 						var xname = "Description";
 						var attr = visual.Attribute(xname);
-						if (attr == null || string.IsNullOrEmpty(attr.Value))
+						if (attr == null || string.IsNullOrEmpty(attr.Value) || attr.Value == DefaultPlaceholder)
 							visual.SetAttributeValue(xname, ApplicationTitle);
 					}
 				}
 
-				// BackgroundColor=""
-				{
-					var xname = "BackgroundColor";
-					var attr = visual.Attribute(xname);
-					if (attr == null || string.IsNullOrEmpty(attr.Value))
-						visual.SetAttributeValue(xname, "transparent");
-				}
```

Comment:
Was this for the tile background? Is this not needed for win10?

Reply:
Oh, sorry. I forgot to mention in the PR that I moved that directly to the templates.

Reply:
![image](https://user-images.githubusercontent.com/59936622/166811555-4cfe368e-05c0-48df-9475-a9228664e373.png)

Reply:
lol

Ok, so maybe have it in both places. In the templates for the UI and then the task to put it back if there is a missing value from older projects.